modules/popup/react/lib/Popup.tsx

```diff
@@ -26,7 +26,7 @@ export interface PopupProps extends React.HTMLAttributes<HTMLDivElement> {
    * The origin from which the Popup will animate.
    * @default {horizontal: 'center', vertical: 'top'}
    */
-  transformOrigin: TransformOrigin;
+  transformOrigin: TransformOrigin | null;
```

Comment:
this allows to pass null in the visual stories and doesn't add the animation allowing for the snap shots to be captured